flexget/plugins/clients/transmission.py

```diff
@@ -286,7 +286,7 @@ class PluginTransmission(TransmissionBase):
                     'password': {'type': 'string'},
                     'action': {
                         'type': 'string',
-                        'enum': ['add', 'remove', 'purge', 'pause', 'resume'],
+                        'enum': ['add', 'remove', 'purge', 'pause', 'resume', 'start now'],
```

Comment:
i'd name this `bypass_queue` to keep it consistent with the original terminology